pkg/strategy/xnav/strategy.go

```diff
@@ -162,24 +174,32 @@ func (s *Strategy) CrossRun(ctx context.Context, _ bbgo.OrderExecutionRouter, se
 	}
 
 	// TODO: if interval is supported, we can use kline as the ticker
-	if _, ok := types.SupportedIntervals[s.Interval]; ok {
+	// if _, ok := types.SupportedIntervals[s.Interval]; ok {
 
-	}
+	// }
 
-	go func() {
-		ticker := time.NewTicker(util.MillisecondsJitter(s.Interval.Duration(), 1000))
-		defer ticker.Stop()
+	if s.Schedule != "" {
```

Comment:
It should be one of "Schedule" or "Interval" ?

Reply:
ok